code/index.html

```diff
@@ -3,15 +3,218 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Business Site</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Flow Movement</title>
+    <link rel="stylesheet" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Petit+Formal+Script&display=swap" rel="stylesheet">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
+  
   <body>
-    <h1>Business name 🌻</h1>
+    <header>
+      <div class="hero">
+        <div class="hero-overlay">
+          <h1>Flow Movement</h1>
+          <h2> - with Gabriella</h2>
+        </div>     
+      </div>
+    </header>
+    
+    <main>
+
+      <!-- Headline for the form -->
+      
+      <div class="form-header">
+        <h3>Create an account.</h3>
+        <h4>Let's get started with your 14 day free trial.</h4>
+      </div>
+      
+
+      <!-- Signup form -->
```

Comment:
I noticed that you've used section comments like <!-- Signup form --> in your code, and I just wanted to say how awesome that is! These comments make the code so much easier to read.